docs/source/getting_started/handling_time.ipynb

```diff
@@ -406,9 +436,17 @@
     "\n",
     "## Secondary Time Index\n",
     "\n",
-    "It is sometimes the case that information in a dataset is updated or added after a row has been created. This means that certain columns may actually become known after the time index for a row. Rather than drop those columns to avoid leaking information, we can create a secondary time index to indicate when those columns become known.\n",
-    "\n",
-    "The [Flights entityset](../generated/featuretools.demo.load_flight.html#featuretools.demo.load_flight) is a good example of a dataset where column values in a row become known at different times. Each trip is recorded in the ``trip_logs`` dataframe, and has many times associated with it."
+    "It is sometimes the case that information in a dataset is updated or added after a row has been created. This means that certain columns may actually become known after the time index for a row. Rather than drop those columns to avoid leaking information, we can create a secondary time index to indicate when those columns become known."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "011c8a13",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "The :func:`Flights <demo.load_flight>` is a good example of a dataset where column values in a row become known at different times. Each trip is recorded in the ``trip_logs`` dataframe, and has many times associated with it."
```

Comment:
This doesn't seem quite right. It reads `The Flights is a good example...` and there is not a working link, at least in my local build.